lang/py/src/avro/schema.py

```diff
@@ -652,7 +654,7 @@ def __init__(self, name, namespace, fields, names=None, schema_type='record',
       NamedSchema.__init__(self, schema_type, name, namespace, names,
                            other_props)
 
-    if schema_type == 'record': 
+    if schema_type == 'record':
```

Comment:
I truly wish github was better about handling white space changes, particularly in places where they are trivial